android/src/main/res/layout/component_bottom_sheet_content.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Does this layout scroll correctly with nested scrolling? I can remember issues with bottom sheets and having to use `NestedScrollView` 🤔